R/eval_forecasts.R

```diff
@@ -70,7 +69,15 @@
 #'   If you want to score the median (i.e. \code{range = 0}), you still
 #'   need to include a lower and an upper estimate, so the median has to
 #'   appear twice.}
-#' @param summarised if \code{TRUE}, only one average score is returned per model
+#' @param by character vector of columns to group scoring by. The default
```

Comment:
is this a breaking change? The default behaviour should really be whatver used to happen or people will get surprising issues. Also I would have though the single model use case should be the default?

Reply:
it shouldn't be. Before you were forced to have a column called "model" and give your model a name even if you only have one model. Now if you put in the same data structure it will by default be scored by the column "model" just as before. You can decide not to have a column called "model" and explicitly set by = "NULL" if you want or you can have more columns to group by.

Reply:
i would have thought the default to no model and make it optional would be just better but its not a big deal